packages/template-retail-react-app/jest-setup.js

```diff
@@ -97,7 +97,6 @@ jest.mock('pwa-kit-runtime/utils/ssr-config', () => {
 
 // TextEncoder is a web API, need to import it
 // from nodejs util in testing environment.
-// This is used in commerce-api/pkce.js
 global.TextEncoder = require('util').TextEncoder
```

Comment:
do we still need this?

Reply:
Yes some tests would fail if we dont have it